website/content/docs/install/windows-service.mdx

```diff
@@ -69,6 +69,26 @@ You have two ways to start the service.
 The service automatically starts up during/after boot, so you don't need to
 launch Nomad from the command-line again.
 
+## Increasing the non-interactive desktop heap size
+
+If you run Nomad as a Windows service and you indend to run many allocations
```

Comment:
```suggestion
If you run Nomad as a Windows service and you intend to run many allocations
```

Reply:
oh english 🫠 
thanks for catching this.